jalali_date/settings.py

```diff
@@ -11,16 +11,11 @@
     'Static': {
         'js': [
             'admin/js/django_jalali.min.js',
-            # or
-            # 'admin/jquery.ui.datepicker.jalali/scripts/jquery.ui.core.js',
-            # 'admin/jquery.ui.datepicker.jalali/scripts/calendar.js',
-            # 'admin/jquery.ui.datepicker.jalali/scripts/jquery.ui.datepicker-cc.js',
-            # 'admin/jquery.ui.datepicker.jalali/scripts/jquery.ui.datepicker-cc-fa.js',
-            # 'admin/js/main.js',
+            'admin/js/prevent_default_listener.min.js'
```

Comment:
Please separate the initial datepicker for specific classes with the source code of the datepicker.
in the old version, we used `main.js` and other js. you should keep and update the `main.js` and ‍`prevent_default_listener.min.js`

Reply:
The reason for the above comment is that users use it for non-admin pages. 

You can create two versions of prevent_default_listener 

- 1. current version
- 2. separate initial datepicker for specific classes

Reply:
I don't get the point of having those js files. After re-adding the class name to the input html element, there is no need to the prevent_default_listener.min.js anymore. The Gregorian date field, and shamsi date field are usable both in admin and the templates. Am I missing sth?

Reply:
I should also update the readme after confusion resolution.